htdocs/libraries/icms/ipf/view/Table.php

```diff
@@ -582,6 +570,38 @@ public function renderForPrint() {
 
 	}
 
+	/**
+	 * Gets from cookie
+	 *
+	 * @param string $fieldName Field name read from cookie
+	 * @param string|null $defaultValue Default value
+	 *
+	 * @return string
+	 */
+	protected function getCookie(string $fieldName, string $defaultValue = null) {
+		$name = 'tbl_' . str_replace('.', '_', $fieldName);
+
+		return $_COOKIE[$name] ?? $defaultValue;
```

Comment:
?? works only in PHP 7.0

Reply:
changed it to return isset($_COOKIE[$name]) ? $_COOKIE[$name] : $defaultValue;